locust/html.py

```diff
@@ -36,7 +38,8 @@ def get_html_report(environment, show_download_link=True):
     requests_statistics = list(chain(sort_stats(stats.entries), [stats.total]))
     failures_statistics = sort_stats(stats.errors)
     exceptions_statistics = []
-    for exc in environment.runner.exceptions.values():
+    exceptions = deepcopy(environment.runner.exceptions)
+    for exc in exceptions.values():
         exc["nodes"] = ", ".join(exc["nodes"])
         exceptions_statistics.append(exc)
```

Comment:
Why not just do something like `exceptions_statistics.append(", ".join(exc["nodes"]))` instead? No need to copy runner.exceptions if `exceptions` is never used again, right?

Reply:
The `exceptions_statistics` must also contain the other fields of the exception dictionary as the html template also needs the other properties:

https://github.com/locustio/locust/blob/60cde73cee8c08bd5c6efb88eb0d511c6c025620/locust/html.py#L66-L80

If you prefer, it could be something like this instead of the for-loop:

```python
exceptions_statistics = [
    {**exc, "nodes": ", ".join(exc["nodes"])}
    for exc in environment.runner.exceptions.values()
]
```

Reply:
I changed it. Looks better now in my opinion.